core/src/test/java/kafka/testkit/KafkaClusterTestKit.java

```diff
@@ -366,7 +366,13 @@ private KafkaClusterTestKit(
         this.baseDirectory = baseDirectory;
         this.faultHandlerFactory = faultHandlerFactory;
     }
+    public MockFaultHandler exposedFatalFaultHandler() {
```

Comment:
https://github.com/apache/kafka/blob/b5a013e4564ad93026b9c61431e4573a39bec766/core/src/test/java/kafka/testkit/KafkaClusterTestKit.java#L592

those handlers are exposed already. Could you please revise the tests to use that existent methods instead?